llama_stack/distribution/build.py

```diff
@@ -82,6 +83,21 @@ def get_provider_dependencies(
 
 
 def print_pip_install_help(config: BuildConfig):
+    if config.external_apis_dir:
+        cprint("Installing external APIs", color="yellow", file=sys.stderr)
+        external_apis = load_external_apis(config)
+        if external_apis:
+            # install the external APIs
+            packages = []
+            for _, api_spec in external_apis.items():
+                if api_spec.pip_packages:
+                    packages.extend(api_spec.pip_packages)
+                    cprint(
+                        f"Installing {api_spec.name} with pip packages {api_spec.pip_packages}",
+                        color="yellow",
+                        file=sys.stderr,
+                    )
+            run_command(["uv", "pip", "install", *packages])
```

Comment:
Not deep in the code, but I don't think that showing a help message should have any side effects like installing dependent packages. This is especially dangerous, as a user looking for help for sure does not want to alter the system. 

Why is this needed here ?